.github/workflows/build.yml

```diff
@@ -0,0 +1,16 @@
+name: Build
+on: 
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: rancher/dapper:v0.6.0
+    steps:
+      - name: Add Git
```

Comment:
The build workflow for pushes is just testing that the code is correctly building. It replicates what drone was doing, running the steps within a container using rancher/dapper:v0.6.0 image. That's the image we are adding git to.

Reply:
You mean `rancher/dapper:v0.6.0` needs git? Otherwise, it should be installed in `Dockerfile.dapper` here https://github.com/rancher/csp-adapter/blob/release/v2.9/Dockerfile.dapper#L7

Reply:
Yep, rancher/dapper needs git to run checkout action and run dapper ci. I'll add a comment on that.